### ISSUE TITLE
pod install fails when cocopods-keys installed

### ISSUE THREAD

**Issue by genadyo** [orta/cocoapods-keys#43]
### Command

```
/Users/genady/.rvm/gems/ruby-2.2.0/bin/pod install
```

### Report

* What did you do?

* What did you expect to happen?

* What happened instead?

### Stack

```
   CocoaPods : 0.36.0
        Ruby : ruby 2.2.0p0 (2014-12-25 revision 49005) [x86_64-darwin14]
    RubyGems : 2.4.5
        Host : Mac OS X 10.10.3 (14D87h)
       Xcode : 6.2 (6C131e)
         Git : git version 2.3.0
Ruby lib dir : /Users/genady/.rvm/rubies/ruby-2.2.0/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 8e7e572ea6e42bb1545f6248b0b33521c936e34c
```

### Plugins

```
cocoapods-keys    : 1.0.1
cocoapods-plugins : 0.4.1
cocoapods-trunk   : 0.6.0
cocoapods-try     : 0.4.3
```

### Podfile

```ruby
platform :ios, '8.0'
inhibit_all_warnings!

pod 'MBProgressHUD'
pod 'VICMAImageView'
#pod 'Reveal-iOS-SDK'

plugin 'cocoapods-keys', {
    :project => "Organizer",
    :keys => [
    ]
}
```

### Error

```
NameError - uninitialized constant Module::Informative
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-keys-1.0.1/lib/plugin.rb:7:in `podspec_for_current_project'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-keys-1.0.1/lib/plugin.rb:66:in `from_string'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-core-0.36.0/lib/cocoapods-core/specification.rb:536:in `from_file'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/external_sources/abstract_external_source.rb:142:in `store_podspec'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/external_sources/abstract_external_source.rb:112:in `block in pre_download'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/user_interface.rb:70:in `titled_section'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/external_sources/abstract_external_source.rb:99:in `pre_download'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/external_sources/downloader_source.rb:13:in `fetch'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/installer/analyzer.rb:347:in `fetch_external_source'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/installer/analyzer.rb:324:in `block (2 levels) in fetch_external_sources'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/installer/analyzer.rb:323:in `each'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/installer/analyzer.rb:323:in `block in fetch_external_sources'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/user_interface.rb:49:in `section'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/installer/analyzer.rb:322:in `fetch_external_sources'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/installer/analyzer.rb:57:in `analyze'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/installer.rb:189:in `analyze'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/installer.rb:111:in `block in resolve_dependencies'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/user_interface.rb:49:in `section'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/installer.rb:110:in `resolve_dependencies'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/installer.rb:90:in `install!'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-keys-1.0.1/lib/plugin.rb:40:in `install!'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/command/project.rb:101:in `run'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/claide-0.8.1/lib/claide/command.rb:312:in `run'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/lib/cocoapods/command.rb:46:in `run'
/Users/genady/.rvm/gems/ruby-2.2.0/gems/cocoapods-0.36.0/bin/pod:44:in `<top (required)>'
/Users/genady/.rvm/gems/ruby-2.2.0/bin/pod:23:in `load'
/Users/genady/.rvm/gems/ruby-2.2.0/bin/pod:23:in `<main>'
```

**Comment by ashfurrow** [comment]
Hmm. I was getting a similar error. I'm not anymore, since I fixed the underlying issue in #44 (use my fork until it's merged if you like – I can help get you set up). 

I'll work on a fix for this in the mean time.

**Comment by ashfurrow** [comment]
Ah, it's already merged :)
